invenio_app_rdm/config.py

```diff
@@ -265,6 +266,10 @@ def files_rest_permission_factory(obj, action):
     'draft_resources': {
         'task': 'invenio_draft_resources.tasks.cleanup_drafts',
         'schedule': timedelta(minutes=60),
+    },
+    'rdm_records': {
+        'task': 'invenio_rdm_records.tasks.update_expired_embargos',
+        'schedule': crontab(minute=2, hour=0),
```

Comment:
Silly question: does it mean at midnight and 2 minutes? Was the minute `2` chosen for some reason?

Reply:
> does it mean at midnight and 2 minutes?

Yes

>  Was the minute `2` chosen for some reason?

No reason at all (Copied from [Zenodo](https://github.com/zenodo/zenodo/blob/master/zenodo/config.py#L181-L185))

Reply:
I *think* is because embargoes will expire right after midnight? e.g. they set a date.